backend/Makefile

```diff
@@ -24,21 +26,49 @@ build-docker: test
 
 ## migrations
 install_goose:
-	go get -u github.com/pressly/goose/v3/cmd/goose
+	$(call install_goose)
 
 goose_up:
-	cd migrations && goose postgres $(PG_STRING) up
+ifeq (, $(shell which goose))
+	$(call install_goose)
+endif
+	cd ./scripts/ && ./goose_up.sh
 
 goose_down:
-	cd migrations && goose postgres $(PG_STRING) down
+ifeq (, $(shell which goose))
+	$(call install_goose)
+endif
+	cd ./scripts/ && ./goose_down.sh
```

Comment:
Maybe we need to add execution rights on all those `shells` we're running, otherwise it will be an issue !? 🤔